### ISSUE TITLE
Latest update shows light theme editor when selecting dark theme

### ISSUE THREAD

**Issue by holly-hacker** [peppy/ppy-jetbrains-theme#3]
In the current version of RustRover (JetBrains's new Rust IDE that is currently in free preview), v0.6 of the theme seems to use the light theme colors for the editor and output window, while using the dark theme colors for the rest of the UI. The plugin worked fine on v0.5, but I can't seem to roll it back.

![Screenshot of v0.6 showing a light-mode editor inside a dark-mode shell](https://github.com/peppy/ppy-jetbrains-theme/assets/13605369/9e3aa00a-3c81-4b5b-8336-a927ff213e7c)

Build info:
```
RustRover 2023.2 EAP
Build #RR-232.9921.62, built on September 15, 2023
Licensed to RustRover EAP user: -snip-
Expiration date: October 15, 2023
Runtime version: 17.0.8+7-b1000.22 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
GC: G1 Young Generation, G1 Old Generation
Memory: 4086M
Cores: 12
Registry:
    ide.experimental.ui=true

Non-Bundled Plugins:
    com.github.catppuccin.jetbrains (2.2.0)
    com.github.madwareru.intellijronremix (0.3.1)
    com.wakatime.intellij.plugin (14.2.2)
    com.nasller.CodeGlancePro (1.7.5)
    com.elliotwaite.godot_theme (1.0.12)
    com.github.copilot (1.3.0.3323)
    com.github.peppy.ppyjetbrainstheme (0.0.6)
```

**Comment by peppy** [comment]
Try restarting your IDE. It might be stuck because of a light-dark transition (I've seen this happen before).

0.5 -> 0.6 changed nothing at all, for what it's worth. It just removed a minver spec.

**Comment by holly-hacker** [comment]
I've tried various combinations of restarting the IDE and switching between themes but this happens consistently. Re-installing the theme also doesn't fix it. Other Dark themes still work fine, including JetBrains Dark Purple which this theme should be based on.

However, I've also tried installing v0.0.5, v0.0.4 and v0.0.2 from the zip releases and they did not seem to work either so I'm guessing something in RustRover updated in the last week that caused the theme to suddenly fail.

I'll ask around to see if other people encounter the same issues and if they do, I'll open a bug report with JetBrains.

**Comment by holly-hacker** [comment]
I asked around and other people don't have this issue in RustRover. By resetting all my local settings the theme works properly again so I assume the issue is with RustRover.

**Comment by holly-hacker** [comment]
Strangely, updating the theme from 0.0.5 to 0.0.6 in Rider 2023.2.2 on my work laptop also caused this. I updated the theme which turned the editor to a light theme for all themes, and after restarting Rider it only shows a white editor for this theme and not others.

I looked through the settings and it seems that the `Editor > Color Scheme > Scheme` was set to a light theme, and changing it to `ppy Dark` fixed the issue. I'm documenting that here in case other people stumble on the same issue.

![image](https://github.com/peppy/ppy-jetbrains-theme/assets/13605369/cc24e718-f06d-40ef-9979-74b9dad5470c)
